net/brave_search_agent/src/brave_search_agent.ts

```diff
@@ -0,0 +1,278 @@
+import { AgentFunction, AgentFunctionInfo, assert, DefaultConfigData } from "graphai";
+
+interface BraveSearchInputs {
+  query: string;
+  search_args?: Record<string, any>;
+}
+
+interface BraveSearchParams {
+  apiKey?: string;
+  debug?: boolean;
+  throwError?: boolean;
+  search_args?: Record<string, any>;
+}
+
+interface BraveSearchResult {
+  title: string;
+  link: string;
+  snippet: string;
+}
+
+type BraveSearchResponse =
+  | {
+      items: BraveSearchResult[];
+    }
+  | {
+      onError: {
```

Comment:
https://github.com/receptron/graphai/blob/main/agents/agent_utils/src/type.ts#L73-L79

Reply:
I fixed it
https://github.com/receptron/graphai-agents/pull/18/commits/6e05e11f45f0c3dbafe0ee911d08ed6ced85f268
https://github.com/receptron/graphai-agents/pull/18/commits/af046ba4e95c7e7c7b82e85c4ae1043c40155e85